openedx_events/learning/data.py

```diff
@@ -296,3 +296,29 @@ class ProgramCertificateData:
     status = attr.ib(type=str)
     url = attr.ib(type=str)
     certificate_available_date = attr.ib(type=datetime, default=None)
+
+
+@attr.s(frozen=True)
+class SpecialExamAttemptData:
+    """
+    Attributes defined for the Open edX Exam downstream effects.
+
+    Note that events that use this data type:
```

Comment:
Do we want another point, like?:
```
* Use the prefix `EXAM_`, which is equivalent to "special exam", but the shorter name is to hopefully avoid confusion because these are the events and exams that would likely be of concern.
```
Feel free to reword.